src/goto-programs/show_symbol_table.cpp

```diff
@@ -51,8 +52,8 @@ void show_symbol_table_brief_plain(
     else
     {
       ptr=get_language_from_mode(symbol.mode);
-      if(ptr==nullptr)
-        throw "symbol "+id2string(symbol.name)+" has unknown mode";
+      INVARIANT_WITH_DIAGNOSTICS(
+        ptr != nullptr, "symbol ", id2string(symbol.name), " has unknown mode");
```

Comment:
There is #2984, which will do away with all this code...

Reply:
Rolled back all changes in this file, hopefully #2984 will clean it adequately.